eclipsePlugin-junit/build.gradle

```diff
@@ -0,0 +1,7 @@
+apply from: "$rootDir/gradle/checkstyle.gradle"
+
+dependencies {
+  compile project(':eclipsePlugin')
+  testCompile 'junit:junit:4.12'
+  testCompile 'org.mockito:mockito-core:2.7.22'
+}
```

Comment:
Wmm, sorry, I couldn't get your point. Do you mean that it's better to keep current design? It is OK for me, then I will trigger unit test by hands, not in CI build.